presto-main/src/main/java/com/facebook/presto/sql/planner/iterative/rule/PickTableLayout.java

```diff
@@ -20,8 +20,10 @@
 import com.facebook.presto.matching.Pattern;
 import com.facebook.presto.metadata.Metadata;
 import com.facebook.presto.metadata.TableLayoutResult;
+import com.facebook.presto.operator.scalar.TryFunction;
```

Comment:
it is strange that planner is using execution code, maybe the method you are using should be extracted to somewhere else (or copied)